frontend/src/cards/MapCard.tsx

```diff
@@ -156,10 +164,28 @@ function MapCardWithKey(props: MapCardProps) {
           .filter(
             (row: Row) => row[props.currentBreakdown] === activeBreakdownFilter
           );
+        const dataForSvi = sviQueryResponse
+          .getValidRowsForField("svi")
+          .filter((row) =>
+            dataForActiveBreakdownFilter.find(({ fips }) => row.fips === fips)
+          );
+
+        const dataForActiveBreakdownFilterWithSvi =
+          dataForActiveBreakdownFilter.map((row) => {
+            const thisCountySviRow = dataForSvi.find(
+              (sviRow) => sviRow.fips === row.fips
+            );
+            return {
+              ...row,
+              svi: thisCountySviRow?.svi,
+              rating: findRating(thisCountySviRow?.svi, true),
+            };
+          });
```

Comment:
should we only do this kind of expensive nested loop filter if it's state or county level? like, 

```
let dataForActiveBreakdownFilter = ...

if (!props.fips.isUsa()) dataForActiveBreakdownFilter = mergeSviRows...
```